src/pcap/cmd/pcap-bosh-cli/main_test.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
Could use a few more tests, but this can also be tackled in a follow-up PR.

Reply:
Yes. We still want to build acceptance tests in a follow-up story. The CLI was planned to be tested in depth there at the latest.